pycue/opencue/util.py

```diff
@@ -27,11 +27,11 @@
 import logging
 import os
 from functools import wraps
+from six import string_types
```

Comment:
I understand existing code isn't consistent on this, but -

It seems like the most reliable way to avoid import problems between py2 and py3 is to use the full-path absolute imports whenever possible, i.e. `import six` and then use `six.string_types` throughout the code.

What do you think?

Reply:
Agreed. I've pushed a new commit that cleans up these imports in this file. I also improved the exception handling to pass the full traceback from grpc calls. @bcipriano PTAL.